samples/subsys/logging/syst/sample.yaml

```diff
@@ -60,3 +64,16 @@ tests:
         - "SYS-T RAW DATA: "
     extra_configs:
       - CONFIG_LOG_MIPI_SYST_USE_CATALOG=y
+  sample.logger.syst.adsp_cavs:
```

Comment:
dont understand why you need this entry and why it is being made specific to this platform, isnt this already covered above? You need to remove the platform_allow from all other scenarios and make all scenarios runnable on all platforms.

Reply:
My understanding has been that when using logging with an adsp_cavs board such as intel_adsp_cavs18, it uses the adsp backend instead of the uart backend, which has necessitated some changes to the .conf files.  I'll dig some more into this.

Reply:
I think I understand more now than I did before. If I have the right mental model, then the sample.logger.syst.adsp_cavs entry is not needed as the custom .conf files under boards/ provide the necessary overrides.

Reply:
Here's what I am finding ... there are various configurations that syst does not support. These include at least 64-bit and big-endian. There may be more. This may be a bit of after-the-fact rationalisation, but I think the original use of platform_allowed instead of integration_platforms may have been done to address those limitations with a simple set of rules.